pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -8,69 +8,143 @@
 permalink: /eks-for-enterprise/pricing/
 
 categories:
-  - title: Feature
+  - title: EKS
+    features:
+      - name: Control plane
+        description: Support for configuring, deploying, and managing the EKS control plane.
+        plans: ["Standard", "Enterprise"]
+      - name: Worker nodes
+        description: "Support for deploying any of the three worker node types: self-managed ASG, EKS Managed Node Groups, and Fargate."
+        plans: ["Standard", "Enterprise"]
+      - name: Core services
+        description: Suite of core Kubernetes applications every EKS cluster should have, including AWS LoadBalancer controller, external-dns, cluster-autoscaler, and more.
+        plans: ["Standard", "Enterprise"]
+      - name: CI/CD
+        description: Support for setting up an infrastructure CI/CD pipeline using traditional CI services (e.g., Jenkins, CircleCI, GitLab CI, etc).
+        plans: ["Standard", "Enterprise"]
+      - name: Basic monitoring
+        description: Support for metric and log aggregation to AWS CloudWatch.
+        plans: ["Standard", "Enterprise"]
+      - name: Basic deployment models
+        description: Support for rolling deployments where a configurable number of service Pods are replaced at a time.
+        plans: ["Standard", "Enterprise"]
+      - name: Basic networking
+        description: Support for AWS VPC based networking for Kubernetes Pods.
+        plans: ["Standard", "Enterprise"]
+      - name: Cluster auto scaling
```

Comment:
"Basic auto scaling"? The hover text provides the explanation of what that includes.